src/gtk/options_dialog.c

```diff
@@ -1272,20 +1171,15 @@ make_proxy_hosts_tab (GtkWidget * notebook)
   gtk_box_pack_start (GTK_BOX (box), hbox, FALSE, FALSE, 0);
   gtk_widget_show (hbox);
 
-#if GTK_MAJOR_VERSION == 1
-  tempwid = gtk_button_new_with_label (_("Add"));
-#else
   tempwid = gtk_button_new_from_stock (GTK_STOCK_ADD);
-#endif
+
   GTK_WIDGET_SET_FLAGS (tempwid, GTK_CAN_DEFAULT);
   gtk_box_pack_start (GTK_BOX (hbox), tempwid, TRUE, TRUE, 0);
   gtk_signal_connect (GTK_OBJECT (tempwid), "clicked",
 		      GTK_SIGNAL_FUNC (add_proxy_host), NULL);
   gtk_widget_show (tempwid);
 
-#if GTK_MAJOR_VERSION == 1
-  tempwid = gtk_button_new_with_label (_("Edit"));
-#elif GTK_MAJOR_VERSION == 2 && GTK_MINOR_VERSION < 5
+#if GTK_MAJOR_VERSION == 2 && GTK_MINOR_VERSION < 5
```

Comment:
I suggest removing this #if as well. GTK+ 2.5 was released in 2004 according to http://ftp.gnome.org/pub/gnome/sources/gtk+/2.5/.